interfaces/inc/c_pal/threadapi.h

```diff
@@ -70,6 +72,13 @@ MOCKABLE_FUNCTION(, THREADAPI_RESULT, ThreadAPI_Join, THREAD_HANDLE, threadHandl
  */
 MOCKABLE_FUNCTION(, void, ThreadAPI_Sleep, unsigned int, milliseconds);
 
+/**
+ * @brief    Gets the current thread Id
+ *
+ * @return    @c a uint32_t value of the current thread Id
+ */
```

Comment:
Frankly these docs are kind of useless for us now.

Reply:
Yeah, I just put it in to be consistent.